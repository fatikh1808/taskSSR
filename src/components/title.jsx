import React from 'react'
import {Col} from 'antd';

export const Title = (children) => {

  return(
    <Col xs={24} sm={24} md={24} lg={24} xl={24}>
        <h1>
          {Object.keys(children).length !== 0 ? children.children : "Articles"}
        </h1>
    </Col>
  )
}