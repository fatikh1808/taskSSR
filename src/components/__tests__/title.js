import React from "react"
import renderer from "react-test-renderer"

import { Title } from "../title"

describe("Paginator", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Title siteTitle="Default Starter"/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})