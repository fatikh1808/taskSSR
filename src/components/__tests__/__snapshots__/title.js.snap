// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paginator renders correctly 1`] = `
<div
  className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-24 ant-col-lg-24 ant-col-xl-24"
  style={Object {}}
>
  <h1 />
</div>
`;
